docs/sovereign/setup.md

```diff
@@ -0,0 +1,221 @@
+# Setup Guide
+
+:::note
+ This guide is a preliminary version and not the final documentation for sovereign chains. It serves as a starting point for setting up a sovereign chain on a local machine.
+:::
+
+This guide will help you set up and deploy contracts on a sovereign chain. Follow these steps carefully to ensure a successful deployment.
+
```

Comment:
Maybe add a message to look at prerequisites before starting the setup

Reply:
done